evenOccurences/evenOccurences.js

```diff
@@ -10,6 +10,23 @@
  *    * console.log(onlyEven); //  4
  *    */
 
-const evenOccurence = (arr) => {
-  // Your code here.
-};
+ const evenOccurence = (arr) => {
+   // Your code here.
+   let numberStorage = {};
+   arr.forEach((number) => {
+     if (numberStorage[number]) {
+       numberStorage[number] = numberStorage[number]+1
+     } else {
+       numberStorage[number] = 1;
+     }
+   })
+   for (let i = 0; i < arr.length; i++) {
```

Comment:
food for thought, someone may ask you to do this without using two for loops. Don't worry about that for now, but maybe keep it in the back of your mind as something to look at later on?